tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+from packaging import version
+
+
+# FIXME: 2.8 does not currently work with netbox-docker, try to get it fixed if possible
+DEFAULT_NETBOX_VERSIONS = "2.9, 2.10" 
```

Comment:
I do still want to see if we can get 2.8 and maybe 2.7 added but that can probably done later on, unless you see some logic issues that may pop up right now.